conda-forge.yml

```diff
@@ -1,6 +1,8 @@
 bot:
   automerge: true
 build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
   osx_arm64: osx_64
```

Comment:
Default behavior is to use emulation for these builds, which is much slower. Have switched Linux arch builds to cross-compile, which is equivalent to native build time

Looks like macOS ARM already does cross-compilation. So the needed pieces are likely already in place with at most minor tweaks